common/openapi.yml

```diff
@@ -3,40 +3,30 @@ info:
   title: Terra Scientific Pipelines Service
   version: 1.0.0
 paths:
-  /status:
-    get:
-      summary: Check status of the service.
-      tags: [ public ]
-      operationId: getStatus
-      security: [ ]
-      responses:
-        200:
-          description: OK
-        500:
-          $ref: '#/components/responses/ServerError'
-        503:
-          $ref: '#/components/responses/SystemStatusResponse'
-  /version:
+  # Admin APIs
+  /api/admin/v1/updatePipelineWorkspaceId/{pipelineName}/{workspaceId}:
+    parameters:
+      - $ref: '#/components/parameters/PipelineName'
+      - $ref: '#/components/parameters/WorkspaceId'
     get:
```

Comment:
should this be a GET? seems like PUT might be more appropriate?

Reply:
(was wondering if we do want a GET method here to just retrieve what workspaceId the service is pointing to)

Reply:
yea thats a great idea, added a GET method and cleaned up the apis a little